src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
@bartonjs this is taken care of in the Jenkins setup. You can see #6139 for some of the other points about conditionally running the tests.